src/Sarif/GitHelper.cs

```diff
@@ -0,0 +1,192 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    public class GitHelper : IDisposable
```

Comment:
>public [](start = 4, length = 6)

Has to be public now because `InsertOptionalDataVisitor` takes the delegate as a parameter, and `InsertOptionalDataVisitor` is a public type, so the parameter type has to be public -- meaning both the delegate declaration (Line 15) and the class have to be public.